README.md

```diff
@@ -1,89 +1,55 @@
-<p align="center">
-  <img src="https://raw.githubusercontent.com/slatedocs/img/master/logo-slate.png" alt="Slate: API Documentation Generator" width="226">
-  <br>
-  <a href="https://github.com/slatedocs/slate/actions?query=workflow%3ABuild+branch%3Amaster"><img src="https://github.com/slatedocs/slate/workflows/Build/badge.svg?branch=master" alt="Build Status"></a>
-</p>
+# Skynet SDK Docs
 
-<p align="center">Slate helps you create beautiful, intelligent, responsive API documentation.</p>
+Documentation for the Skynet SDKs in various languages.
 
-<p align="center"><img src="https://raw.githubusercontent.com/slatedocs/img/master/screenshot-slate.png" width=700 alt="Screenshot of Example Documentation created with Slate"></p>
+## Status of SDKs
 
-<p align="center"><em>The example above was created with Slate. Check it out at <a href="https://slatedocs.github.io/slate">slatedocs.github.io/slate</a>.</em></p>
+### Supported functions
 
-Features
-------------
+The following is a list of functionality and whether it is supported by each language. Cells marked by `**` are not yet documented.
```

Comment:
I'm wondering whether it's useful to have the entire signature.. in which case the format is not ideal?
Perhaps the format works for the time being. My biggest issue with the SDKs is not in naming but rather response objects, I think all SDKs should have methods that wrap the siad API endpoints, and thus return similar responses as if you were to execute a curl command...

I would then wrap those in more human readable / friendly functions. This would allow SDK users to access our API in a 1-1 fashion, which is always good I think.

Reply:
Raised https://github.com/NebulousLabs/skynet-docs/issues/6